bin/update_hostname_validator.php

```diff
@@ -0,0 +1,130 @@
+<?php
```

Comment:
Should this file to have a shebang and chmod +x?

It's what I expect for any bin file